src/transformers/models/auto/modeling_auto.py

```diff
@@ -888,6 +905,17 @@ class AutoModelForVisualQuestionAnswering(_BaseAutoModelClass):
 )
 
 
+class AutoModelForDocumentQuestionAnswering(_BaseAutoModelClass):
+    _model_mapping = MODEL_FOR_DOCUMENT_QUESTION_ANSWERING_MAPPING
+
+
+AutoModelForDocumentQuestionAnswering = auto_class_update(
+    AutoModelForDocumentQuestionAnswering,
+    head_doc="document question answering",
+    checkpoint_for_example="impira/layoutlm-doc-qa",
```

Comment:
We prefer to have checkpoints for pipeline to be under the huggingface organization, rather than third-party ones, as these could break things.

Reply:
The two above are from `google` and `dandelin`. I'm very happy to change this, but am not sure there's a layoutlm-based alternative that's published by `huggingface` and fine-tuned for question answering. If I'm missing it, would you mind pointing me to it? Alternatively, I could remove it altogether. Let me know what you all prefer.